data/campaigns/Eastern_Invasion/scenarios/12_Evacuation.cfg

```diff
@@ -1,391 +1,951 @@
-            #po: "I'm not charging gold — I want protection! Everywhere I go, I see orcs, undead. It isn't safe around here!"
-            message= _ "I ain’t charging gold — I wants protecshun! Everywhere I go, I see orcs, undead. ’T’ain’t safe ’round ’ere!"  # wmllint: no spellcheck
-        [/message]
-        [message]
-            speaker=Gweddry
-            message= _ "Very well, we’ll protect you from the undead. But we can’t help you until we get off this island. Now let’s go!"
-        [/message]
-        [message]
-            speaker=Engineer
-            #po: "Deal. I can blow her [it] up once I get to that signpost over there. That's where my equipment is. But everyone not over there on that side when it blows will be killed by the orcs for sure!"
-            message= _ "Deal. I c’n blow’er up once I get to that signpost ov’r dere. That’s where my eq’pment is. But ev’ryone not ov’r theah on that side when it blows w’ll be killed by the orcs fa’ sure!"   # wmllint: no spellcheck
-        [/message]
-        [item]
-            image=scenery/signpost.png
-            x,y=8,20
-        [/item]
-    [/event]
+    [/side]
+    {SILENTLY_LIMIT_LEADER_MOVES 4 1}
+    {STARTING_VILLAGES_AREA 4 0-99 19-99 0}
+    {LIMIT_CONTEMPORANEOUS_RECRUITS 4 "Orcish Grunt"       2}
+    {LIMIT_CONTEMPORANEOUS_RECRUITS 4 "Orcish Warrior"     3} # includes the starting loyal
+    {LIMIT_CONTEMPORANEOUS_RECRUITS 4 "Orcish Archer"      2}
+    {LIMIT_CONTEMPORANEOUS_RECRUITS 4 "Orcish Crossbowman" 4} # includes the starting loyal
+    {LIMIT_CONTEMPORANEOUS_RECRUITS 4 "Orcish Assassin"    2}
 
     [event]
-        name=moveto
+        name=side 1 turn end
         first_time_only=no
-        [filter]
-            x,y=8,20
-            id=Engineer
-        [/filter]
-        [message]
-            speaker=Engineer
-            #po: "So, do you want me to blow up the bridge yet, Captain?"
-            message= _ "So, d’ya want me to blow up der bridge yet, Cap’n?" # wmllint: no spellcheck
-        [/message]
-        [message]
-            speaker=Gweddry
-            message= _ "Hmm..."
-            [option]
-                label= _ "Yes, blow it up."
-                [command]
-                    [message]
-                        speaker=Engineer
-                        #po: "All right! Blasting time!"
-                        message= _ "All right! Blast’n time!" # wmllint: no spellcheck
-                    [/message]
-                    [sound]
-                        name=fuse.ogg
-                    [/sound]
-                    [delay]
-                        time=2000
-                    [/delay]
-                    [sound]
-                        name=explosion.ogg
-                    [/sound]
+        {RESET_SIDE_AI 2,3,4 offensive 0.8 0.25}
+
+        # drakes don't fly across the river
+        {MODIFY_SIDE_AI (3) ([avoid]
+        x,y=0-5,0-99
+    [/avoid])}
+
+    {VARY_AI_BY_SCHEDULE_WINTER 2,3,4}
+[/event]
```

Comment:
Regarding CI check failure, it looks like everything including and after this line should be indented a level.